docassemble/ILAO/data/questions/shared-basic-questions.yml

```diff
@@ -193,11 +193,6 @@ subquestion: |
 ---
 # added 11-1-2021 to customize email ALDocumentBundle forms functionality
 ---
-generic object: ALDocumentBundle
-template: x.get_email_copy
-content: |
-  Get a copy of your forms by email
----
```

Comment:
Removed this because there's a newer version of the variable set at the end of this file ("Send a copy of your forms to your email")